Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -96,7 +96,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>v140</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>
```

Comment:
I will change it to Unicode. It seems to matter only for which version of Win32 APIs are called. Since we don't expect anyone to be running Vireo in the future on any WinOS older than Win-NT, I think we can go with Unicode. I will fix both.